src/demo/js/script.js

```diff
@@ -62,12 +62,11 @@ let preview = {
       label.setAttribute("data-property", property);
       // color picker
       const input = document.createElement("input");
-      input.type = "color";
-      input.className = "param";
+      input.className = "param jscolor";
       input.id = property;
       input.name = property;
-      input.value = "#dd2727"; // default red color
       input.setAttribute("data-property", property);
+      input.setAttribute("data-jscolor", "{ preset: 'customPreset' }");
```

Comment:
Set format to `hexa` and use a default color instead of white.

`value` can be an optional parameter to the `addProperty` function:

```js
   addProperty: function (property, value = "#DD2727FF") {
	 ...
```

If `value` will be added, you can remove this from the window load event listener:

```js
	preview.addProperty(key);
    paramInput = document.querySelector(`#${key}`);
    paramInput.value = val;
```

And put this instead:

```js
    preview.addProperty(key, val);
```

Suggestion to replace the preset:

```suggestion
      input.setAttribute("data-jscolor", "{ format: 'hexa' }");
      input.value = value;
```